password-store-otp.el

```diff
@@ -67,7 +67,10 @@
 Clear previous password from kill ring.  Pointer to kill ring is
 stored in `password-store-kill-ring-pointer'.  SECRET is cleared
 after `password-store-timeout' seconds."
-  (password-store-clear)
+  ; the signature of `password-store-clear' changed:
```

Comment:
Change suggestion:

```
; The signature of `password-store-clear' momentarily changed
; between versions 2.1.0 and 2.1.2:
; - breaking change: https://git.zx2c4.com/password-store/commit/?id=3a108277637fad89869d904eea963aeb8f45b20d
; - fix:             https://git.zx2c4.com/password-store/commit/?id=e74a1c738f7cda65c7a308e30e8d122f853d6f70
```

Reply:
I'm not sure it's worth keeping complex code to workaround an issue that disappeared. I vote for closing my PR.